static/form.js

```diff
@@ -306,8 +306,24 @@ function checkInputRadius(inputId, errorMsg){
   const inputElement = document.getElementById(inputId);
```

Comment:
why is the `inputId` of the element holding the radius passed in as an argument to this function, whereas the `"pathWidth"` element id below is hard-coded?  in fact it's hard coded in more than once place in this file!